### PR TITLE
[SW-482] Use SDK13 in latest transactions & fix account discovery

### PR DESCRIPTION
**Pull request by pstachel**

### CODE REVIEW

**Review by subhod-i — APPROVED** [review]

File: src/popup/components/Modals/AccountImport.vue [hunk 1]
```diff
@@ -99,9 +102,10 @@ export default defineComponent({
       store.commit('setBackedUpSeed');
       props.resolve();
       setTimeout(async () => {
-        await getSdk();
-        store.dispatch('accounts/hdWallet/discover');
-      }, 100);
+        watchUntilTruthy(() => isSdkReady.value).then(
```
Comment (peronczyk) [inline src/popup/components/Modals/AccountImport.vue:105]: Why not awaiting the `getSdk` ?

**Review by peronczyk — APPROVED** [review]

File: src/popup/components/Modals/AccountImport.vue (hunk 1 above)
Comment (CedrikNikita) [reply src/popup/components/Modals/AccountImport.vue:105]: https://github.com/aeternity/superhero-wallet/pull/2135#discussion_r1246485890

Comment (peronczyk) [reply src/popup/components/Modals/AccountImport.vue:105]: We'll resolve this in the future :)

Comment (pstachel) [reply src/popup/components/Modals/AccountImport.vue:105]: That change doesn't reflect what happened properly - `getSdk` is not working here but previously we were only checking if sdk is ready. Usage of `getSdk` here was a mistake as it wasn't working

**Pull request merged**